modules/Bio/Otter/Lace/Defaults.pm

```diff
@@ -669,7 +669,7 @@ __DATA__
 
 
 [client]
-url=http://172.22.10.136:8004/cgi-bin/otter
+url=http://172.22.10.173:8004/cgi-bin/otter
```

Comment:
Could we please just use _otter-server.ebi.ac.uk_ here? The HTTP modules we use in Otter client handle name resolution fine, I've just checked.

Reply:
Changed.